src/Components/Home.js

```diff
@@ -0,0 +1,65 @@
+import { useState, useEffect } from 'react';
+import { Link } from 'react-router-dom';
+import { useSelector, useDispatch } from 'react-redux';
+import { FaGlobe } from 'react-icons/fa';
+import { fetchCountries } from '../redux/slices/countriesSlice';
+
+function HomePage() {
+  const { countries } = useSelector((state) => state.countries);
+  const [searchTerm, setSearchTerm] = useState('');
+  const dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchCountries());
+  }, [dispatch]);
+
+  const handleSearch = (event) => {
+    setSearchTerm(event.target.value);
+  };
+
+  let filteredCountries = countries;
+  if (searchTerm) {
+    filteredCountries = countries.filter(
+      (country) => country.name.toLowerCase().includes(searchTerm.toLowerCase()),
+    );
+  }
+
+  return (
+    <section className="home-page">
+      <div className="flex items-center justify-between p-4">
+        <h1 className="text-2xl font-bold text-gray-800">
+          <FaGlobe className="inline-block mr-2" />
+          Countries
+        </h1>
+        <div className="flex items-center justify-between">
+          <input
+            type="text"
+            placeholder="Search for a country..."
+            className="bg-gray-200 rounded-full px-4 py-2 outline-none"
+            value={searchTerm}
+            onChange={handleSearch}
+          />
+        </div>
+      </div>
+      <div className="grid grid-cols-3 gap-4 p-4">
+        {filteredCountries.map((country) => (
+          <div key={country.alpha2Code} className="border rounded-lg">
+            <Link to={`/details/${country.alpha2Code}`}>
+              <img
+                src={country.flag}
+                alt={country.name}
+                className="w-full h-52 mx-auto border-4 border-gray-200 object-cover"
+              />
+              <p className=" bg-white rounded text-center text-lg">
+                {country.name}
+              </p>
+            </Link>
+          </div>
+        ))}
+      </div>
+    </section>
+
```

Comment:
- [ ] Great work using Tailwind CSS for this project. Kindly design this section of the Home Page to look identical to that of the template provided. In the template, all items are listed two in a row while on yours, they are listed three in a row. It would be nice for you to follow the design guidelines for this project. Also, kindly follow the design guideline for individual cards. 👍 💻 

### Your Design

![image](https://user-images.githubusercontent.com/77106962/220473654-49dd534d-fbba-4527-b68a-4a55e342523e.png)

### Behance Template

![image](https://user-images.githubusercontent.com/77106962/220473826-0f75a3fe-b046-43fd-a8f0-cd57a8fbfff1.png)